### PR TITLE
[FEATURE] Add Page TSconfig provider for sites and sets

### PR DESCRIPTION
**Pull request by linawolf**
Releases: main

### CODE REVIEW

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 1]
```diff
@@ -75,6 +75,72 @@ TYPO3 v11 and v12 by importing the content of this file with the API function
...
+===========================
+
+..  versionadded:: 13.1
+    Page TSconfig can be included on a per site level.
+
+Site Page TSconfig is loaded from :file:`page.tsconfig` if placed next to
+site configuration file :file:`config.yaml` and is scoped to pages within that
+site.
+
+This way sites and sets can ship page TSconfig without the need for database
```
Comment (garvinhicking) [inline Documentation/UsingSetting/PageTSconfig.rst:91]:
```suggestion
This way sites and sets can ship Page TSconfig without the need for database
```

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 2]
```diff
@@ -75,6 +75,72 @@ TYPO3 v11 and v12 by importing the content of this file with the API function
...
+entries or by polluting global scope. Dependencies can expressed via site sets,
```
Comment (garvinhicking) [inline Documentation/UsingSetting/PageTSconfig.rst:92]:
```suggestion
entries or by polluting global scope. Dependencies can be expressed via site sets,
```

Comment (garvinhicking) [inline Documentation/UsingSetting/PageTSconfig.rst:92]: What does "polluting global" mean in this case? Adding conditional expressions into a central tsconfig file for "switches"? If thats the case, maybe we can reword it a bit (or else remove the "polluting" part if we can't pinpoint what that means?)

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 3]
```diff
@@ -75,6 +75,72 @@ TYPO3 v11 and v12 by importing the content of this file with the API function
...
+allowing for automatic ordering and deduplication.
+
+See also :ref:`Feature: #103437 - Introduce site sets <changelog:feature-103437-1712062105>`
+
+..  todo: Link site sets once they are documented
+
+.. _include-static-page-tsconfig-per-site-example:
+
+Example: load page TSconfig from the site set and the site
```
Comment (garvinhicking) [inline Documentation/UsingSetting/PageTSconfig.rst:101]:
```suggestion
Example: load Page TSconfig from the site set and the site
```

**Review by garvinhicking — APPROVED** [review]
Nice, thanks! I believe we use "Page TS..." woth uppercase, suggested a few changes to unify that. Else, keep all lowercase please if I am wrong ☺️

File: Documentation/UsingSetting/PageTSconfig.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/UsingSetting/PageTSconfig.rst]: I actually copied this from the changelog entry but that is probably what they mean?

File: Documentation/UsingSetting/PageTSconfig.rst (hunk 3 above)
Comment (linawolf) [reply Documentation/UsingSetting/PageTSconfig.rst:101]: it should be lowercase actually

File: Documentation/UsingSetting/PageTSconfig.rst (hunk 2 above)
Comment (garvinhicking) [reply Documentation/UsingSetting/PageTSconfig.rst]: OK. Before we phrase it wrong, let's leave it at that. It's not recommended to "pollute" anymore, so we must not explain it :D

**Review by garvinhicking — APPROVED** [review]
Thanks for the lowercasing-notice. I'll try to burn it into my brain.

**Pull request merged**